### ISSUE TITLE
Server Error 500 in `scd/test_operation_references_error_cases.py::test_op_already_exists_v15`

### ISSUE THREAD

**Issue by svishnu-flyanra** [interuss/dss#569]
**Current Behaviour of test  `scd/test_operation_references_error_cases.py::test_op_already_exists_v15`** :- 
Throws an error 400 claiming that more than there is an overlapping circle and polygon in the extents object. 
Looking at the file [op_request_1_v_15.json](https://github.com/interuss/dss/blob/scd_api_update/monitoring/prober/scd/resources/op_request_1_v15.json#L15), there is a circle and a polygon in one location. This violates the spec [operational_intent_references > extents > volume](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/astm-utm/Protocol/master/utm.yaml#operation/createOperationalIntentReference). 

**Changes**: To resolve this the polygon outline or circle outline is removed from [op_request_1_v_15.json](https://github.com/interuss/dss/blob/scd_api_update/monitoring/prober/scd/resources/op_request_1_v15.json#L15). 

**Expected Behaviour**: This should resolve the issue or give an error code in the 400s but 

**Actual Behaviour**: we get an `error 500 internal server error`. We get the same error irrespective of removing the circle or the polygon. 

**Steps to reproduce**:

**Command**: ` test/docker_e2e.sh scd/test_operation_references_error_cases.py::test_op_already_exists_v15`

**Test output**:

```
=================================== FAILURES ===================================
__________________________ test_op_already_exists_v15 __________________________

ids = <function ids.<locals>.<lambda> at 0x7f5f85b70d30>, scd_api = '0.3.17'
scd_session = <monitoring.monitorlib.infrastructure.DSSTestSession object at 0x7f5f85ba3460>

    @for_api_versions(scd.API_0_3_17)
    @default_scope(SCOPE_SC)
    def test_op_already_exists_v15(ids, scd_api, scd_session):
      with open('./scd/resources/op_request_1_v15.json', 'r') as f:
        req = json.load(f)
      resp = scd_session.put('/operational_intent_references/{}'.format(ids(OP_TYPE)), json=req)
>     assert resp.status_code == 200, resp.content
E     AssertionError: b'{
E         "error": "Internal server error E:57e3d79f-f03b-498d-910d-f786eabaea53",
E         "code": 13,
E         "message": "Internal server error E:57e3d79f-f03b-498d-910d-f786eabaea53",
E         "error_id": "E:32740a52-0542-4707-a9e7-d215fbf66521"
E       }'
E     assert 500 == 200
E      +  where 500 = <Response [500]>.status_code

scd/test_operation_references_error_cases.py:245: AssertionError
```

**Comment by svishnu-flyanra** [comment]
The same error shows up in 

- `scd/test_operation_references_error_cases.py::test_op_already_exists_v15`
- `scd/test_operation_references_error_cases.py::test_missing_conflicted_operation_v15`
- `scd/test_operation_references_error_cases.py::test_op_repeated_requests_v15`

However the last one (`test_op_repeated_requests_v15`) fails gives an error `500` with original file too.

**Comment by BenjaminPelletier** [comment]
Fixed by #571 I believe.
